metrics/mixin/alerts/storage-client.libsonnet

```diff
@@ -0,0 +1,69 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'storage-client-alerts.rules',
+        rules: [
+          {
+            alert: 'StorageClientHeartbeatWarn',
+            expr: |||
+              time() - %(clientCheckinWarnSec)d > ocs_storage_client_last_heartbeat > 0
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s)' % $._config.clientCheckinWarnSec,
+              description: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s). Lossy network connectivity might exist' % $._config.clientCheckinWarnSec,
+              severity_level: 'warning',
+            },
+          },
+          {
+            alert: 'StorageClientHeartbeatCrit',
+            expr: |||
+              time() - %(clientCheckinWarnSec)d > ocs_storage_client_last_heartbeat > 0
+            ||| % $._config,
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s)' % $._config.clientCheckinWarnSec,
+              description: 'Storage Client ({{ $labels.storage_consumer_name }}) heartbeat missed for more than %d (s). Lossy network connectivity might exist' % $._config.clientCheckinCritSec,
+              severity_level: 'critical',
+            },
+          },
+          {
+            # divide by 1000 here removes patch version
+            alert: 'StorageClientIncompatibleCluster',
+            expr: |||
+              floor(abs((ocs_storage_client_cluster_version > 0) - ignoring(storage_consumer_name) group_left() (ocs_storage_provider_cluster_version > 0)) / 1000) > %(clientClusterMinorVerDiff)d
```

Comment:
do you really want
floor(abs((ocs_storage_client_cluster_version > 0)
instead of:
floor(abs(ocs_storage_client_cluster_version) > 0)